src/coreclr/jit/inlinepolicy.cpp

```diff
@@ -94,8 +94,16 @@ InlinePolicy* InlinePolicy::GetPolicy(Compiler* compiler, bool isPrejitRoot)
         return new (compiler, CMK_Inlining) ProfilePolicy(compiler, isPrejitRoot);
     }
 
-    // Use the default policy by default
-    return new (compiler, CMK_Inlining) DefaultPolicy(compiler, isPrejitRoot);
+    const bool useExtendedDefaultPolicy = JitConfig.JitExtendedDefaultPolicyModel() != 0;
+
+    if (!useExtendedDefaultPolicy || (!isPrejitRoot && compiler->opts.IsReadyToRun()))
```

Comment:
I find the second clause easier to parse written the other way. Also we should probably use the more general check for prejitting

```suggestion
    if (!useExtendedDefaultPolicy || (compiler->opts.jitFlags->IsSet(JitFlags::JIT_FLAG_PREJIT) && !isPrejitRoot))
```